packages/block-library/src/archives/editor.scss

```diff
@@ -1,3 +1,7 @@
 .block-editor ul.wp-block-archives {
 	padding-left: 2.5em;
 }
+
+.block-editor .wp-block-archives__label {
+	display: block;
```

Comment:
Should this not also be `display: block;` on the front-end?

This note applies to the categories block as well.

Reply:
At the time I think I wanted these to be as least opinionated as possible in themes but I can make these changes to render the controls in the same manner on the front end.

Reply:
So why is it part of the editor styles in the first place?